.nvmrc

```diff
@@ -1 +1 @@
-18
\ No newline at end of file
+lts/hydrogen
```

Comment:
Why not 18.17 just for consistency. Might work better with renovate too?

Reply:
Sure thing, updated. I originally picked ts/hydrogen so the file doesn't need to be updated with every node 18 release.